ProcessLib/ThermalTwoPhaseFlowWithPP/ThermalTwoPhaseFlowWithPPLocalAssembler-impl.h

```diff
@@ -0,0 +1,469 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+/**
+* Common convenitions for naming:
+* X_gas_nonwet           mass fraction of gas component(e.g air) in nonwetting phase
+* (gas component doesn't include water vapor, same for the following)
+* x_gas_nonwet           molar fraction of gas component in nonwetting phase
+* x_vapor_nonwet         molar fraction of vapor in nonwetting phase
+* p_vapor_nonwet         water vapor pressure
+* p_gas_nonwet           partial pressure of gas component
+* rho_mol_nonwet         molar density of nonwetting phase
+* rho_mol_water          molar density of water
+* rho_water              mass density of water
+* rho_nonwet_gas         mass density of gas component in the nonwetting phase
+* rho_nonwet_vapor       mass density of vapor in the nonwetting phase
+* rho_nonwet             mass density of the nonwetting phase
+* rho_wet                mass density of wetting pahse
+* rho_solid              mass density of the solid phase
```

Comment:
Minor: Here you choose to use rho_solid for density, yet at other places you use density_solid.